verification/src/compiler/compilers.rs

```diff
@@ -44,3 +46,9 @@ impl<T: Fetcher> Compilers<T> {
         Ok(output)
     }
 }
+
+impl<T: VersionList> Compilers<T> {
```

Comment:
It would be better to implement `VersionList` for `Compilers`

Reply:
don't think that `impl<T: VersionList> VersionList for Compilers<T>` looks readable now) but I don't mind, okay